src/main/java/ua/danit/final_project/services/WorkCommentServiceImpl.java

```diff
@@ -2,10 +2,12 @@
 
 import org.joda.time.DateTime;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.springframework.http.ResponseEntity'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=112&issues=AWXJ9xVd_eEZPtUmsYm3&open=AWXJ9xVd_eEZPtUmsYm3)

[ISSUE_KEY]: AWXJ9xVd_eEZPtUmsYm3